From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Logan Bussell <loganbussell@microsoft.com>
Date: Fri, 13 May 2022 16:14:14 -0700
Subject: [PATCH] Add NoWarn for NU1507

This warning occurs when using NuGet central package management without defining
any Package Source Mappings.

Workaround for: https://github.com/dotnet/razor-compiler/issues/242
---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index a57125f5..019ac376 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -55,6 +55,7 @@
     <!-- Ensure API docs are available. -->
     <NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
     <NoWarn Condition=" '$(BuildingInsideVisualStudio)' == 'true' ">$(NoWarn);0105</NoWarn>
+    <NoWarn>$(NoWarn);NU1507</NoWarn>
 
     <!-- For local builds, don't make missing XML docs a fatal build error, but still surface so we have visibility into undocumented APIs. -->
     <WarningsNotAsErrors Condition=" '$(ContinuousIntegrationBuild)' != 'true' ">$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
